src/graphql/main/schema.graphql

```diff
@@ -27,6 +28,14 @@ interface Account {
   wallets: [Wallet!]!
 }
 
+type AccountInformation {
+  firstName: String!
+  lastName: String
+  otherField: Int
+  otherField2: Boolean
+  transactionsCallback: String
+}
+
```

Comment:
Is this temp too?

The main GraphQL schema should not have any static values for account information. I'm not sure if there's a way to send back dynamic values though. Best I can think of is an array of arrays where everything is a string (but there's a `type` value in there somewhere:

`[ [name, type, value], [name, type, value], ...]`

Reply:
yep, if we remove default.yaml fields this will be removed too

Reply:
👍🏽 everything else look good.

Reply:
will the `schema.graphql` dynamically expand when passing a custom yaml @dolcalmi ?

Reply:
(I think that would make sense and would be great, because it would make the custom fields discoverable from the API)

Reply:
> will the `schema.graphql` dynamically expand when passing a custom yaml @dolcalmi ?

yes, it will